app/controllers/RouterController.scala

```diff
@@ -73,12 +73,13 @@ trait RouterController extends FrontendController with Actions {
   val account = AuthenticatedBy(authenticationProvider = RouterAuthenticationProvider, pageVisibility = AllowAll).async { implicit user => request => route(user, request) }
 
   def route(implicit authContext: AuthContext, request: Request[AnyContent]): Future[Result] = {
-
-    val ruleContext = RuleContext(authContext)
-
+    val ruleContext = RuleContext(None)
```

Comment:
agreed to create the rule context using a function so that it can be checked in unit tests